tensorflow/compiler/xla/service/gpu/ir_emitter_unnested.cc

```diff
@@ -737,19 +767,21 @@ Status IrEmitterUnnested::HandleCopy(HloInstruction* copy) {
     thunk_sequence_->emplace_back(BuildKernelThunk(copy));
     VLOG(3) << "Emitting tiled 0-2-1 transposition";
     constexpr int64 tile_size = 32;
+    constexpr int64 num_rows = 4;
     int64 num_tiles = EmitTranspose021Tiled(
         GetIrArray(*(copy->operand(0)))
             .CastToShape(reduced_input_shape, &ir_builder_),
         GetIrArray(*copy).CastToShape(reduced_output_shape, &ir_builder_),
-        tile_size, &ir_builder_);
-    UpdateLaunchDimensions(LaunchDimensions(num_tiles, tile_size), LastThunk(),
-                           ir_emitter_context_->llvm_module());
+        tile_size, num_rows, &ir_builder_);
+    UpdateLaunchDimensions(LaunchDimensions(num_tiles, num_rows * tile_size),
+                           LastThunk(), ir_emitter_context_->llvm_module());
     return Status::OK();
   }
 
   return IrEmitter::HandleCopy(copy);
 }
 
+
```

Comment:
Nit, in another commit, you removed a bunch of blank lines, not sure why adding this one here.

Reply:
Hi, you seem to be referring to an outdated file. This empty line is removed in commit 3ef49faa94775f3995c5eb9e20256513d86f382a.

Reply:
Oh, I see -- I was reading the patch queue in the reverse order.  That makes more sense now.  :)

I don't know if TF as a whole has a preference, but for my part I'd prefer to rebase -i so that we don't commit changes that we then undo.  That is, small commits are good, but only if they're incrementally correct.  (And if it's a pain to do that, certainly in this case I'd be totally fine squashing this whole thing into one commit.)

But it's not a big deal.